kerl

```diff
@@ -599,8 +599,12 @@ maybe_patch() {
     release=$(get_otp_version "$2")
     case "$1" in
         Darwin)
+            xcode_version=$(/Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -version | head -n 1 | awk '{print $2}')
             # Catalina and clang require a "no-weaks-import" flag during build
-            if [[ $(uname -r) == "19"* ]] && [[ "$release" -lt 23 ]] && [[ $2 != "22.3.1" ]]; then
+            if [[ $(uname -r) == "19"* ]] && \
+               [[ "$release" -lt 23 ]] && \
+               [[ $(printf "$2\n22.3.1" | sort | head -n 1) != "22.3.1" ]] && \
+               [[ $(printf "$xcode_version\n11.4" | sort | head -n 1) == "11.4" ]]; then
```

Comment:
This `sort` is not semver sort. Not sure if that is an issue here but if a less ambiguous tool can be used here that'd be great.

Reply:
@fenollp thank you for the feedback, looks like the system sort has a --version-sort that would solve it, it also handles rc versions
```
-V, --version-sort
             Sort version numbers.  The input lines are treated as file names in form PREFIX VERSION SUFFIX, where SUFFIX matches the
             regular expression "(.([A-Za-z~][A-Za-z0-9~]*)?)*".  The files are compared by their prefixes and versions (leading zeros
             are ignored in version numbers, see example below).  If an input string does not match the pattern, then it is compared
             using the byte compare function.  All string comparisons are performed in C locale, the locale environment setting is
             ignored.
```

Reply:
Oh I was under the impression `sort -V` wasn't available on osx. Nice.